content/config/nav.yaml

```diff
@@ -198,6 +198,15 @@ tutorials:
           url: /tutorials/smart-contracts/erc20-token/
         - title: Troubleshoot smart contracts
           url: /tutorials/smart-contracts/sc-common-issues/
+    - title: Run a Light Client in your browser
+      url: /tutorials/light-clients/
+      pages:
+        - title: Download the browser extension
+          url: /tutorials/light-clients/substrate-connect/
+        - title: Connect with a well-known relay-chain
+          url: /tutorials/light-clients/production-relays/
+        - title: Connect with a parachain
+          url: /tutorials/light-clients/production-parachains/
```

Comment:
@lisa-parity you approve of the structure here? We can refactor if you think it is needed, and/or move to the "right place" in nav